MSALiOSB2C/ViewController.swift

```diff
@@ -228,123 +196,94 @@ class ViewController: UIViewController, UITextFieldDelegate, URLSessionDelegate
+            }
+            }.resume()
         
     }
     
     @IBAction func signoutButton(_ sender: UIButton) {
-        
-        /**
-         The way B2C knows what actions to perform for the user of the app is through the use of `Authority URL`,
-         a URL indicating a directory that MSAL can use to obtain tokens. In Azure B2C
-         it is of the form `https://<instance/tfp/<tenant>/<policy>`, where `<instance>` is the
-         directory host (e.g. https://login.microsoftonline.com), `<tenant>` is a
-         identifier within the directory itself (e.g. a domain associated to the
-         tenant, such as contoso.onmicrosoft.com), and `<policy>` is the policy you wish to
-         use for the current user flow. The policy we are using here is the `kSignupOrSigninPolicy`
-         *as that is the policy we originally acquired the token with*. It is very important that all actions
-         against a token acquired by a policy maintains the user of that policy on each call.
-         */
-        
-         let kAuthority = String(format: kEndpoint, kTenantName, kSignupOrSigninPolicy)
-        
         do {
-            
             /**
+             Removes all tokens from the cache for this application for the provided account
              
-             Initialize a MSALPublicClientApplication with a given clientID and authority
-             
-             - clientId:     The clientID of your application, you should get this from the app portal.
-             - authority:    A URL indicating a directory that MSAL can use to obtain tokens. In Azure B2C
-             it is of the form `https://<instance/tfp/<tenant>/<policy>`, where `<instance>` is the
-             directory host (e.g. https://login.microsoftonline.com), `<tenant>` is a
-             identifier within the directory itself (e.g. a domain associated to the
-             tenant, such as contoso.onmicrosoft.com), and `<policy>` is the policy you wish to
-             use for the current user flow.
-             - error:       The error that occurred creating the application object, if any, if you're
-             not interested in the specific error pass in nil.
+             - account:    The account to remove from the cache
              */
             
-            let application = try MSALPublicClientApplication.init(clientId: kClientID, authority: kAuthority)
-            
-            /**
-             Removes all tokens from the cache for this application for the provided user
-             
-             - user:    The user to remove from the cache
-             */
+            let thisAccount = try self.getAccountByPolicy(withAccounts: application.allAccounts(), policy: kSignupOrSigninPolicy)
             
-            let thisUser = try self.getUserByPolicy(withUsers: application.users(), forPolicy: kSignupOrSigninPolicy)
+            if let accountToRemove = thisAccount {
```

Comment:
nit: should we log an error if we don't find an account here?

Reply:
done :)